bciers/apps/registration/app/components/operations/registration/OptedInOperationPage.tsx

```diff
@@ -0,0 +1,39 @@
+import { UUID } from "crypto";
+import { validate as isValidUUID } from "uuid";
+import OptedInOperationForm from "apps/registration/app/components/operations/registration/OptedInOperationForm";
+import { optedInOperationSchema } from "apps/registration/app/data/jsonSchema/operationRegistration/optedInOperation";
+import { getOptedInOperationDetail } from "@bciers/actions/api";
+
+const OptedInOperationPage = async ({
+  operation,
+  step,
+  steps,
+}: {
+  operation: UUID;
+  step: number;
+  steps: string[];
+}) => {
+  let formData;
+  if (operation && isValidUUID(operation)) {
+    formData = await getOptedInOperationDetail(operation);
+  }
+
+  if (formData && "error" in formData) {
+    return (
+      <div>
+        <p>Sorry, Something went wrong. Please try again later!</p>
+      </div>
+    );
+  }
```

Comment:
We already have built in error handling via our `error.tsx` files. I suggest throwing an error here instead ie:

```suggestion

  if (formData && "error" in formData) {
    throw new Error(
      "We couldn't find your operation information. Please ensure you have been approved for access to this operation.",
    );
  }
```

<img width="1365" alt="Screenshot 2024-08-22 at 1 09 16 PM" src="https://github.com/user-attachments/assets/4d326029-ffd8-4ab3-b2d6-f1c7d77bb4a4">